src/Wallabag/CoreBundle/Resources/views/themes/material/Developer/index.html.twig

```diff
@@ -14,7 +14,8 @@
 
                 <ul>
                     <li><a href="{{ path('developer_howto_firstapp') }}">{{ 'developer.how_to_first_app'|trans }}</a></li>
-                    <li><a href="{{ path('nelmio_api_doc_index') }}">{{ 'developer.full_documentation'|trans }}</a></li>
+                    <li><a href="http://doc.wallabag.org/en/v2-documentation-api/developer/api.html">{{ 'developer.full_documentation'|trans }}</a></li>
+                    <li><a href="{{ path('nelmio_api_doc_index') }}">{{ 'developer.list_methods'|trans }}</a></li>
```

Comment:
Why don't you change that text in baggy too?
Also, if you do not plan to use `developer.full_documentation`, you can remove it from the translations files.